### PR TITLE
fixed linters errors

### PR DESCRIPTION
**Pull request by richie1988**
fixed linters errors

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,109 +3,12 @@ const hamburger = document.querySelector('.hamburger');
...
-        menu.classList.add('showMenu');
-        closeIcon.style.display = 'block';
-        menuIcon.style.display = 'none';
-    }
-}
+const toggleMenu = () => {
+  const showMenu = menu.classList.toggle('showMenu');
+  closeIcon.style.display = showMenu ? 'block' : 'none';
+  menuIcon.style.display = showMenu ? 'none' : 'block';
+};
```
Comment (omar-muhamad) [inline index.js:10]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=mobile%20menu%20disappears.-,When%20the%20user%20clicks%20(or%20taps)%20any%20of%20the%20mobile%20menu%20options%2C%20a%20correct%20part%20of%20the%20page%20is%20displayed.,-We%20included%20a), When I click `contact` in mobile menu option it doesn't display the contact form section, please fix that.

File: mobilepopup.css [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    text-align: center;
+    justify-content: center;
+    display: flex;
+    padding: 0 10px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    color: hsl(20, 3%, 23%);
+  }
```
Comment (omar-muhamad) [inline mobilepopup.css:43]: - [ ] For the line below the mobile menu options, please increase the width of it to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 3]
```diff
@@ -1,1160 +1,1151 @@
...
-    top: 1rem;
-    padding: 4px;
-    cursor: pointer;
-    margin-left: 13rem;
+  z-index: 100;
+  top: 1rem;
+  padding: 4px;
+  cursor: pointer;
+  margin-left: 13rem;
 }
```
Comment (KanzaTahreem) [inline style.css:37]: - [OPTIONAL] The cross button is behaving as expected on `375px` However, when I am increasing the width, the button is misplaced ⚠😓 please make sure that it is always placed at the end, could you please fix this? ✨

image

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @richie1988 

Good job implementing the suggested changes 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Your README looks good, well done!
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕

_Check out the comments under review_


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**